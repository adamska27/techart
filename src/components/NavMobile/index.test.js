import React from "react";
import renderer from "react-test-renderer";

import Component from "./index";

describe("NavMobile", () => {
  it("should match the snapshot", () => {
    let component = renderer.create(<Component />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
