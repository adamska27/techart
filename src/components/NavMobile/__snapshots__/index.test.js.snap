// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMobile should match the snapshot 1`] = `
<div
  className="sc-bdVaJa dGcPrY"
  transform="transform: translateX(-100%)"
>
  Je suis la nav mobile
</div>
`;
